lib/interpreters/periodic_visits.rb

```diff
@@ -510,7 +510,8 @@ def generate_relations_on_periodic_vehicles(vrp, list)
     private
 
     def get_original_values(original, options)
-      [original.attributes.keys + options.keys - [:unavailable_work_day_indices] + [:unavailable_days]].flatten.each_with_object({}) { |key, data|
+      old_keys = [:unavailable_work_day_indices, :unavailable_visit_day_indices]
```

Comment:
why do we need this? since these fields don't exists `original.attributes.keys` shouldn't return them unless it is a "dump" issue again?